src/main/java/org/json/simple/ItemList.java

```diff
@@ -40,7 +41,9 @@ public List getItems(){
 	}
 	
 	public String[] getArray(){
-		return (String[])this.items.toArray();
+		Object[] objectArray = this.items.toArray();
+		String[] stringArray = Arrays.copyOf(objectArray, objectArray.length, String[].class);

```

Comment:
I think it's not necessary to save the object array to a separate variebla, we can return with it right away.